content/en/synthetics/_index.md

```diff
@@ -8,73 +8,49 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
```

Comment:
For the further reading section, I'm suggesting, in order:
https://www.datadoghq.com/blog/introducing-synthetic-monitoring/
https://www.datadoghq.com/blog/browser-tests/
https://www.datadoghq.com/blog/test-creation-best-practices/
https://docs.datadoghq.com/synthetics/guide/ (not many of them, but we're planning to add a lot more)